docs/plugins/index.rst

```diff
@@ -66,6 +66,7 @@ like this::
    inline
    ipfs
    keyfinder
+   kodinfo
```

Comment:
Nice! Could you also add this to `plugins/index.rst` with a human-readable summary of the plugin?